onnxruntime/core/common/profiler.cc

```diff
@@ -2,11 +2,153 @@
 // Licensed under the MIT License.
 
 #include "profiler.h"
+#include <cmath>
+
+#ifdef USE_CUDA
+#include <cupti.h>
```

Comment:
So, when USE_CUDA is defined, the core runtime's code(which is in onnxruntime.dll) will depend on CUDA DLLs?

Reply:
@RyanUnderhill FYI
will we need to re-do https://github.com/microsoft/onnxruntime/commit/6a889ee8bf72f34f44b8a4642728b7bd83f61255 ?